ansible/idr-02-services.yml

```diff
@@ -12,3 +12,4 @@
 
 ## Search services
 - import_playbook: idr-searchengine.yml
+- import_playbook: idr-run-searchengine-container.yml
```

Comment:
Without reading the content, I find these names make it hard to understand the intent of each playbook. See below for a proposal